ml/db/README.md

```diff
@@ -0,0 +1,445 @@
+# INSERTS
+
+## RANKING RELATION
+
+### Insert new ranking into ranking relation
+```sql
+-- Template
+INSERT INTO ranking (name) VALUES ('<newRankName');
+
+-- Example
+INSERT INTO ranking (name) VALUES ('Hydrogen');
+```
+
+## PLATFORM RELATION
+
+### Insert new platform into platform relation
+```sql
+-- Template
+INSERT INTO platform (name) VALUES ('<platformNameUppercase>');
+
+-- Example
+INSERT INTO platform (name) VALUES ('FRIDGE');
+```
+
+## SEASON RELATION
+
+### Insert new season into season relation
+```sql
+-- Template
+INSERT INTO season (name) VALUES ('<seasonNameShorthand>');
+
+-- Example
+INSERT INTO season (name) VALUES ('FS21');
+```
+
+## GAMEMODE RELATION
+
+### Insert new gamemode into gamemode relation
+```sql
+-- Template
+INSERT INTO gamemode (name) VALUES ('<gamemodeName>');
+
+-- Example
+INSERT INTO gamemode (name) VALUES ('HideAndSeek');
+```
+
+## GAMETYPE RELATION
+
+### Insert new gametype into game gametype relation
+```sql
+-- Template
+INSERT INTO gametype (name) VALUES ('<gametypeName>');
+
+-- Example
+INSERT INTO gametype (name) VALUES ('Elevens');
+```
+
+## TEAM RELATION
+
+### Insert team data into team relation
+
+```sql
+-- Template
+INSERT INTO team (clubname, score) VALUES ('<clubname>', <score>);
+
+-- Example
+INSERT INTO team (clubname, score) VALUES ('Federal Aerial Administration', 7);
+
+INSERT INTO team (clubname, score) VALUES (null, 6);
+```
+
+## PLAYER RELATION
+
+### Insert player data into player relation
+
+```sql
+-- Template
+INSERT INTO player VALUES ('<uniqueID>', '<playerUsername>', <teamNum>, <platformNum>, <playerRanking>, <mvpBool>, <proBool>);
+
+-- Example
+INSERT INTO player VALUES ('21928347342', 'Novarchite', 1, 0, 16, true, false);
+
+INSERT INTO player VALUES ('82348098233', 'Chicken', 2, 1, 16, false, false);
+```
+
+## REPLAY RELATION
+
+### Insert replay data into replay relation
+
+```sql
+-- Template
+INSERT INTO replay (id, name, uploader, recorder, uploaddate, playingdate, duration, overtime, arena, blueteam, orangeteam, season, ranked, avgrank, gamemode, gametype)
+VALUES ('<ballchasingReplayID>', '<storedReplayName>', '<uploaderUsername>', '<recorderUsername>', '<uploadDateText>', '<playingDateText>', <replayDurationSecondsInteger>, <overtimeDurationSecondsInteger>, '<arenaName>', <blueteamTeamID>, <orangeteamTeamID>, <seasonID>, <rankedBool>, <averagePlayerRank>, <gamemodeID>, <gametypeID>)
+
+-- Example
+INSERT INTO replay (id, name, uploader, recorder, uploaddate, playingdate, duration, overtime, arena, blueteam, orangeteam, season, ranked, avgrank, gamemode, gametype)
+VALUES ('89831272', -- ballchasingReplayID
+        'Hat Trick Assists', -- storedReplayName
+        'Oakerinos', -- uploaderUsername
+        'Chicken', -- recorderUsername
+        '2022-04-17', -- uploadDateText
```

Comment:
The date will most likely be formatted in the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601), which will include timezone information. Just wanted to keep that in mind.